src/Log/README.md

```diff
@@ -0,0 +1,116 @@
+# Log
+
+The Log package is a [psr/log](https://github.com/php-fig/log) compatible logger implementation using WordPress actions to log messages using [Monolog](https://github.com/Seldaek/monolog) behind the scenes.
+
+As this uses WordPress actions, it doesn't couple any logger to your code, and simply won't fire any log
+messages if no logger is configured.
+
+## Quick Start
+
+Register `Log_Definer.php` and `Log_Subscriber.php` in Square One Core.php. Call the Log Actions below
+in your code.
+
+## Log Action Definition
+
+```php
+/**
+ * @param string $logger_hook Use on of the constants defined in Log_Actions.
+ * @param string $message The message to log.
+ * @param mixed[] $additional_data_to_log Additional data to log. 
+ */
+do_action( string $logger_hook, string $message, array $additional_data_to_log );
+```
+
+## Example Usage
+
+```php
+do_action( \Tribe\Libs\Log\Log_Actions::INFO, 'Starting connection...' );
+
+// ... more logic
+
+if ( $status !== 200 ) {
+    do_action( \Tribe\Libs\Log\Log_Actions::ERROR, 'Invalid status returned from API.', [ 'status' => $status ] );
+}
+
+```
+## Log Action List
```

Comment:
Rename Log Levels, so it's clear these are bottom up selection.

Reply:
These are technically actions, as the constants actually map to WordPress actions of `tribe/log/emergency` `tribe/log/alert` etc...

Reply:
Yeah I get that, but they relate to a log level you are logging a message as. So maybe:

```suggestion
## Log Level Action List
```

Reply:
I like it.